src/app/learn/resources/index.html

```diff
@@ -0,0 +1,9 @@
+<app-nav></app-nav>
+<stache pageTitle="Resources">
+
+  <stache-page-summary>
+    <p>
+      Resources is an excellent place to provide links to further documetnation that help support your product or service.
```

Comment:
Maybe: "Use resources to provide additional links, supplemental information, and any other pertinent details that support your service."